nvflare/utils/components_utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_classes_table_static():
+    from nvflare.app_common.abstract.aggregator import Aggregator
+    from nvflare.app_common.abstract.shareable_generator import ShareableGenerator
```

Comment:
Can these abstract classes be used directly?

Reply:
removed these and add the popular PTFileModelPersistor and PTFileModelLocator in the list.